rasa/nlu/extractors/duckling_http_extractor.py

```diff
@@ -113,7 +120,7 @@ def _duckling_parse(self, text, reference_time):
                 "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"
             }
             response = requests.post(
-                self._url() + "/parse", data=payload, headers=headers
+                self._url() + "/parse", data=payload, headers=headers,timeout=self._timeout()
```

Comment:
```suggestion
                self._url() + "/parse", data=payload, headers=headers, timeout=self.component_config.get("timeout")
```

You can simply get the value from the dict here.

Reply:
Addressed this review comment.